src/templates/chat/license.html

```diff
@@ -1,23 +1,21 @@
 <div class="ose game-license">
-  <p class="ose game-license">
-    Unofficial Old-School Essentials on Foundry VTT <!-- BEGIN TEXT REQUIRED BY LICENSE -->requires <strong><em>Old-School Essentials</em></strong>.<!-- END TEXT REQUIRED BY LICENSE --> Find the game at <a href="https://necroticgnome.com">Necrotic Gnome's website</a>.
+  <p>
+    Simple, robust rules compatible with decades of B/X and OSR content. <a href="https://necroticgnome.com">See the website for full information about Old-School Essentials.</a>
+  </p>
+  <p>
+    Old-School Essentials is a trademark of Necrotic Gnome, permitted for use under contract.
+  </p>
+  <p>
+    Report your issues on our
+    <a href="https://github.com/vttred/ose/issues/new">Github issue tracker</a>
+    or by using the Bug Reporter module in Foundry VTT.
   </p>
   <p>
-    <!-- BEGIN TEXT REQUIRED BY LICENSE -->
-    This third party product is not affiliated with or approved by Necrotic Gnome.
-
-    Old-School Essentials is a trademark of Necrotic Gnome. The trademark and Old-School Essentials logo are used with permission of Necrotic Gnome, under license. <!-- ADDITIONAL TEXT REQUESTED BY NECROTIC GNOME --> Permission to use Old-School Essentials in the ruleset title granted by Necrotic Gnome. <!-- END ADDITIONAL TEXT -->
-    <!-- END TEXT REQUIRED BY LICENSE -->
+    Open Source Acknowledgements
```

Comment:
I feel like this should probably be a heading.

Reply:
I understand that, but I'll have to shrink it with css or go directly to h4/h5/h6